bin/build_auth_release.yaml

```diff
@@ -0,0 +1,8 @@
+# Used for automated builds triggered on Git tags pushed matching the regex:
+#   \d+\.\d+\.\d+
+
+steps:
+- name: 'gcr.io/$PROJECT_ID/istio-builder'
+  args: [ 'bash', './bin/push-docker.sh', '-h', "gcr.io/$PROJECT_ID", '-t', "$TAG_NAME" ]
```

Comment:
Hard coded the project name.  Will add docker.io once the credential work is done.